exporter/exporterhelper/internal/queuebatch/persistent_queue.go

```diff
@@ -238,6 +253,85 @@ func (pq *persistentQueue[T]) Offer(ctx context.Context, req T) error {
 	return pq.putInternal(ctx, req)
 }
 
+// necessary due to SpanContext and SpanContextConfig not supporting Unmarshal interface,
+// see https://github.com/open-telemetry/opentelemetry-go/issues/1819.
+type spanContext struct {
+	TraceID    string
+	SpanID     string
+	TraceFlags string
+	TraceState string
+	Remote     bool
+}
+
+func spanContextFromContext(ctx context.Context) spanContext {
+	sc := trace.SpanContextFromContext(ctx)
+	return localSpanContextFromTraceSpanContext(sc)
+}
```

Comment:
Used only once, this func does not seem to simplify code since also can be written in one line and used only once.